app/forms/supplementary/AdditionalDeclarationType.scala

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms.supplementary
+
+import play.api.data.Forms.text
+import play.api.data.{Form, Forms, Mapping}
+import play.api.libs.json.Json
+
+case class AdditionalDeclarationType(
+  value: String // 1 upper case alphabetic character
+)
```

Comment:
Do you need to have a method `toMetadataProperties`?

Reply:
Not here. AdditionalDeclarationType together with DispatchLocation are parts of DeclarationType. Alone, they have no mapping.
DeclarationType only is going to be converted to properties.